### PR TITLE
update scripts for generated files

### PR DESCRIPTION
**Pull request by jduimovich**
Update the copy scripts to reference the generated templates directory.
- copy-to-tssc-templates.sh copys the directory and contents so if/when we add techdocs it will update automatically
- ci-test only copies the specific CI files to avoid overwriting ther files in the test repos

### CODE REVIEW

File: ci-test.sh [hunk 1]
```diff
@@ -9,18 +9,22 @@ if [ $TEST_REPO_ORG == "redhat-appstudio" ]; then
...
 echo "Update Jenkins file in $BUILD and $GITOPS" 
-echo "Jenkins to reuse the Github repo" 
-cp Jenkinsfile $BUILD/Jenkinsfile   
-cp Jenkinsfile.gitops $GITOPS/Jenkinsfile
+echo "Jenkins is able to reuse the same Github repo as github actions" 
+GEN_SRC=generated/source-repo
+GEN_GITOPS=generated/gitops-template   
+
+cp $GEN_SRC/jenkins/Jenkinsfile $BUILD/Jenkinsfile  
+cp $GEN_GITOPS/jenkins/Jenkinsfile $BUILD/Jenkinsfile
```
Comment (simonbaird) [inline ci-test.sh:24]:
```suggestion
cp $GEN_GITOPS/jenkins/Jenkinsfile $GITOPS/Jenkinsfile    
```

**Review by simonbaird — COMMENTED** [review]
I didn't test it, but lgtm other than the one mismatch I noticed.

**Review by chmeliik — APPROVED** [review]
The `hack/copy-to-tssc-templates` changes work :+1: 

Can't comment on `ci-test.sh` because I haven't come across it yet and I don't know what it's for

**Comment by simonbaird** [comment]
Lgtm. The formatter has a couple of complaints, but let's fix it later.

**Pull request merged**